api/src/main/java/org/openmrs/module/eptsreports/reporting/library/queries/DsdQueries.java

```diff
@@ -0,0 +1,12 @@
+package org.openmrs.module.eptsreports.reporting.library.queries;
+
+public class DsdQueries {
+
+  public static String getPatientsEnrolledOnGAAC() {
+    String query =
+        "SELECT gm.member_id FROM gaac g INNER JOIN gaac_member gm ON g.gaac_id=gm.gaac_id "
```

Comment:
Waiting for Pinki's response on this as well. In the meantime, I will try to set up all the databases to my local machine.

Reply:
Oh in case you're not aware, we have the big QA server up now. So you can use that for running your report across all the partners

https://docs.google.com/document/d/15R95410vqUiQ-deLwAiEfuxHBMXGruTTxmw8IlJrFMw/edit

Reply:
@psmatsinhe  ran the scripts and found that all the databases have the gaac table. 
`arielndlavele has gaac\n arieltenga has gaac\n ccsmascarada has gaac\n ccsmascaradapvls has gaac\n ccspepfar has gaac\n egpafmabil has gaac\n egpafsalela has gaac\n fghgile has gaac\n fghgurue has gaac\n icapakumi has gaac\n icapdb has gaac\n icapmarrere has gaac\n icapnacala has gaac\n icapnmp has gaac\n icappsiquiatrico has gaac\n openmrs has gaac\n`